flexget/plugins/search_t411.py

```diff
@@ -0,0 +1,248 @@
+
+              2D, 3D-Converti-Amateur, 3D-Converti-Pro, 3D-Natif
+    """
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'category': {'type': 'string'},
+            'sub_category': one_or_more(
+                {'type': 'string', 'enum': list(SUB_CATEGORIES)}
+            ),
+        },
+        'required': ['username', 'password'],
+        'additionalProperties': False
+    }
+
+    @plugin.internet(log)
+    def search(self, task, entry, config=None):
+        """
+        Search torrent in private torrent tracker t411
+        """
+
+        if not isinstance(config, dict):
+            config = {}
+
+        category = config.get('category')
+
+        if category == 'Serie-TV':
+            sub_categories_dict = SUB_CATEGORIES.copy()
+            sub_categories_dict.update(SUB_CATEGORIES_SERIES)
+
+        if category in list(CATEGORIES):
+            category = CATEGORIES[category]
+
+        sub_categories = config.get('sub_category')
+        if not isinstance(sub_categories, list):
+            sub_categories = [sub_categories]
+
+        filter_url = ''
+        if isinstance(category, int):
+            filter_url = '?cat=%s' % str(category)
+
+            if sub_categories[0] is not None:
+                sub_categories = [sub_categories_dict[c] for c in sub_categories]
+                filter_url = filter_url + '&' + '&'.join([quote_plus('term[%s][]' % c[0]).
+                                                          encode('utf-8') + '=' + str(c[1])
+                                                          for c in sub_categories if c != 0])
+        entries = set()
+        auth_handler = t411Auth(config.get('username'), config.get('password'))
+        for search_string in entry.get('search_strings', [entry['title']]):
+            query = normalize_unicode(search_string)
+            url_search = ('/torrents/search/' +
+                          quote_plus(query.encode('utf-8')) +
+                          filter_url)
+
+            response = requests.get(T411_BASE_URL + url_search, auth=auth_handler)
+            res_json = response.json()
+
+            for torrent in res_json['torrents']:
+                entry = Entry()
+                entry['title'] = torrent['name']
+                entry['url'] = T411_BASE_URL + '/torrents/download/%s' % torrent['id']
+                entry['torrent_seeds'] = torrent['seeders']
+                entry['torrent_leeches'] = torrent['leechers']
+                entry['search_sort'] = torrent_availability(entry['torrent_seeds'],
+                                                            entry['torrent_leeches'])
+                entry['content_size'] = torrent['size']
+                entry['download_auth'] = auth_handler
+                entries.add(entry)
+
+
+            return sorted(entries, reverse=True,
```

Comment:
Shouldn't this be unindented one level? You return immediately after querying for the first search string.